.github/workflows/generator_dart.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
       # the inputs should be pinned, including the generator version and the
       # googleapis SHA.
       - name: Regenerate existing clients
-        run: go run github.com/googleapis/librarian/cmd/sidekick@v0.1.2-0.20250812135937-eb593463c88c refreshall -project-root dart
+        run: go run github.com/googleapis/librarian/cmd/sidekick@v0.1.2-0.20250826011555-c0c043b7a0a6 refreshall -project-root dart
```

Comment:
Given a commit in googleapis/librarian, how do I know the right reference value to use here (i.e., `20250826011555-c0c043b7a0a6`)?

Also, do we want to keep the two librarian repo references here - in generator_dart.yaml and sdk.yaml - in sync?

Reply:
> Given a commit in googleapis/librarian, how do I know the right reference value to use here (i.e., `20250826011555-c0c043b7a0a6`)?

That we just figured out today:

https://github.com/googleapis/google-cloud-rust/blob/main/doc/contributor/howto-guide-generated-code-maintenance.md#special-cases

> Also, do we want to keep the two librarian repo references here - in generator_dart.yaml and sdk.yaml - in sync?

Good question ... let's see how it goes by trying to keep them in sync. If it becomes a problem we can change our minds.